test/e2e/framework/volume/fixtures.go

```diff
@@ -663,7 +663,8 @@ func InjectContent(f *framework.Framework, config TestConfig, fsGroup *int64, fs
 // generateWriteCmd is used by generateWriteBlockCmd and generateWriteFileCmd
 func generateWriteCmd(content, path string) []string {
 	var commands []string
-	commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + path}
+	// flush data so volume can be immediately used as a data source
+	commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + path + " && sync " + path}
```

Comment:
this will run in the agnhost image that has sync.exe so I think this works, the latest version of our e2e tests for Windows of PD CSI succeeded with sync https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver#ci-windows-2019-provider-gcp-compute-persistent-disk-csi-driver

Reply:
@mauriciopoppe I remember we added sync command in other PR? Is this PR still needed?

Reply:
yes I think it's needed, the ones we added were for the snapshottable tests, these ones affect tests in testsuites/volumes.go and testsuites/provisioning.go

Reply:
@msau42 @jingxu97 I think we need this line, some provisioning e2e tests are failing and I suspect that it's because of us not executing sync, I opened https://github.com/kubernetes/kubernetes/pull/109924